starknet_py/utils/decorators/code_examples/code_examples.py

```diff
@@ -0,0 +1,51 @@
+import re
+from typing import Any, Tuple
+
+from starknet_py.constants import ROOT_PATH
+
+
+def add_code_examples(original_class: Any) -> Any:
+    base_class = original_class.__base__
+
+    file_name, file_content = _extract_file_properties(original_class.__name__)
+
+    for method_name, method in original_class.__dict__.items():
+        stripped_method_name = method_name.strip("_")
+        if f"""docs: {stripped_method_name}_start""" in file_content:
```

Comment:
Maybe it would be easier to track the functions if it was something like `docs-start: method_name`?